representation/lsk.metta

```diff
@@ -14,22 +14,24 @@
    (let*
    (
     ($negatedSubtree (insertAbove $subtree (mkNode NOT)))
-    ($reducedSubtree $negatedSubtree)  ;;;after the reduct integration, the reduction step will be applied here 
+    ; (() (println! (NegatedSubtree : $negatedSubtree)))
+    ($reducedSubtree (cleanTree $negatedSubtree))  ;;;after the reduct integration, the reduction step will be applied here 
+    ; (() (println! (ReducedSubtree: $reducedSubtree))) 
     ($childrenOfTarget (getChildrenById $tree (mkNodeId $target)))
    )
    (if (List.contains $subtree $childrenOfTarget) 
        (let $idOfSubtree (getSubtreeId $tree (mkNodeId $target) $subtree 0)
-           ($tree (mkLSK (mkDiscKnob (mkKnob (updateID $idOfSubtree)) (mkMultip 3) (mkDiscSpec 1) (mkDiscSpec 1) Nil)))
+           ($tree (mkLSK (mkDiscKnob (mkKnob $idOfSubtree) (mkMultip 3) (mkDiscSpec 1) (mkDiscSpec 1) Nil)))
```

Comment:
Any particular reason for removing the `updateID` function call? It was handling a weird edge case when it was implemented?

Reply:
I am pretty sure the leading zeros are a result of the removal of this function too.